Library/Homebrew/test/dev-cmd/extract_spec.rb

```diff
@@ -0,0 +1,235 @@
+describe "brew extract", :integration_test do
```

Comment:
Integration tests are comparatively really slow. As a result I think it's sufficient to have a single test that verifies the happy path. If you wish to add more tests consider refactoring some logic out of `cmd` and into something that can be unit tested (but I think a single test is fine here for now).

Reply:
This was addressed 👍